conf/application.conf

```diff
@@ -66,15 +65,8 @@ tai {
   taxFreeAmountComparison.enabled = true
   cyPlus1EstimatedPay.enabled = true
   webChat.enabled = true
-}
-
-metrics {
```

Comment:
Still need this?

Reply:
There were duplicated metrics so we deleted one of them